markdownlint/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:11-alpine
+
+ENV MARKDOWNLINT_VERSION=0.15.0
+
+RUN npm install -g markdownlint-cli@${MARKDOWNLINT_VERSION}
+
```

Comment:
Usually it's a good practice to provide a `CMD` or `ENTRYPOINT`

Reply:
Sure, fixed, what are the rules for this? I mean, we have Dockerfiles with and without CMD or ENTRYPOINT even here in `kiwicom/dockerfiles`, so there seems to be a bit of inconsistency.

Reply:
Yep, the basic rule is that the docker image usually should be able to work without having to override the command or entrypoint, that's why it's a good practice to provide one.

Regarding the inconsistency... yes, unfortunately nothing is enforced, so that's why is like this.

Reply:
I see, thank you :-)